app/jobs/appointment_notification/worker.rb

```diff
@@ -98,6 +99,7 @@ def send_message(notification, communication_type)
       notification.status_sent!
     end
     logger.info("notification #{notification.id} communication_type=#{communication_type} sent")
+    response
```

Comment:
Looks like you needed this in order to get the `sent_message` from the worker. Nice.